### PR TITLE
Fix docs details for app deployment docs

### PR DESCRIPTION
**Pull request by kulmann**
Some more fixes in app deployment docs

### CODE REVIEW

File: docs/deployments/oc10-app.md [hunk 1]
```diff
@@ -74,6 +79,10 @@ There are a few config values which need to be set in order for ownCloud Web to
 }
 ```
 
+{{< hint info >}}
+Please note that the `server` url needs to have a trailing slash.
+{{< /hint >}}
+
```
Comment (micbar) [inline docs/deployments/oc10-app.md:85]: not needed anymore, thanks to `\rtrim`

Comment (kulmann) [reply docs/deployments/oc10-app.md:85]: That only applies to the `web.baseUrl`. config.json still needs the trailing slash for the `server` url. On the other hand, it makes more sense to enforce it when loading the server url. Will make a separate PR.

**Comment by kulmann** [comment]
Screenshot 2020-12-17 at 12 17 24

@micbar  this happens when you provide a `server` url in the config.json without a trailing slash.

**Comment by micbar** [comment]
ah, config.json
Sorry, i thought it was config.php

**Comment by kulmann** [comment]
@micbar https://github.com/owncloud/web/pull/4536 fixes it. Will adapt this PR accordingly.

**Comment by pascalwengerter** [comment]
@kulmann definitely not pressing, but this needs a quick rebase and should be good to merge :)

**Comment by kulmann** [comment]
Has been solved in other PRs in the meantime. Closing here.

**Pull request closed without merging**